tests/flow/test_concurrent_query.py

```diff
@@ -165,7 +164,9 @@ def test_04_concurrent_delete(self):
         for i in range(CLIENT_COUNT):
             t = threads[i]
             t.join()
-            assert(assertions[i])
+
+        # Exactly one thread should have successfully deleted the graph.
+        assert(assertions.count(True) == 1)
```

Comment:
With the changes here and in the graph delete op, only one thread can successfully delete the graph. I think this makes more sense.